content/en/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -30,21 +30,35 @@ large Kubernetes clusters.
 
 ## Percentage of Nodes to Score
 
-Before Kubernetes 1.12, Kube-scheduler used to check the feasibility of all
-nodes in a cluster and then scored the feasible ones. Kubernetes 1.12 added a
-new feature that allows the scheduler to stop looking for more feasible nodes
-once it finds a certain number of them. This improves the scheduler's
-performance in large clusters. The number is specified as a percentage of the
-cluster size. The percentage can be controlled by a configuration option called
-`percentageOfNodesToScore`. The range should be between 1 and 100. Larger values
-are considered as 100%. Zero is equivalent to not providing the config option.
-Kubernetes 1.14 has logic to find the percentage of nodes to score based on the
-size of the cluster if it is not specified in the configuration. It uses a
-linear formula which yields 50% for a 100-node cluster. The formula yields 10%
-for a 5000-node cluster. The lower bound for the automatic value is 5%. In other
-words, the scheduler always scores at least 5% of the cluster no matter how
-large the cluster is, unless the user provides the config option with a value
-smaller than 5.
+To improve scheduling performance, the kube-scheduler can stop looking for
+feasible nodes once it has found enough of them. In large clusters, this saves
```

Comment:
@sftim , I think you could 1-2 sentences as background content instead of diving right in.
A possible intro sentence:

```
To improve the scheduling performance of large clusters, the kube-scheduler relies upon the 
configuration option, `percentageOfNodesToScore`. This setting determines a threshold for scheduling nodes in your cluster.
In large clusters, creating a threshold [saves time?] compared to a naive approach that considers [the possibility of] scheduling every node.

### Setting the threshold

The `percentageOfNodesToScore` option [flag?] accepts whole numeric values between 1 and 100.
...
```

How is this configuration set? As a flag on the kube-scheduler? Is there an example in the docs?
Why would you want to set the value to 0? Or would you restart the scheduler with no configuration?

Reply:
Thanks for the feedback on this @kbhawkey.